README.md

```diff
@@ -801,15 +801,13 @@ See the [AUTHORS][] file for author contact information.
 License
 -------
 
-Bedrock and all Bedrock modules are:
-
-    Copyright (c) 2011-2024 Digital Bazaar, Inc.
-
 You can use Bedrock for non-commercial purposes such as self-study, research,
-personal projects, or for evaluation purposes. Many Bedrock libraries (including
-the core) are licensed as open source under the [BSD-3-Clause](LICENSE.md). However,
-some components use a custom non-commercial source available license: see the
-[LICENSES/LicenseRef-bedrock.md](LICENSES/LicenseRef-bedrock.md) file for details.
+personal projects, or for evaluation purposes. Bedrock libraries (including
+the core) are licensed as open source under the [BSD-3-Clause](LICENSE).
```

Comment:
- Why remove the copyright notice?  Isn't that fairly standard practice to include?

Concerns from https://github.com/digitalbazaar/bedrock-express/pull/56#discussion_r1488852333 remain.
- Why explain, only in part, what you can use code for?  Leave that to the license text.
- This doesn't mention what license this code is using.
- It is making a blanket statement about the entire project, but there could be per-file or `.reuse/dep5` exceptions.

Reply:
Mostly it's the collision between the various methods in place for licensing the README.

The shift is toward explicit per-file licensing vs. project-wide licensing (which is often incorrect, inadequate, etc.).

There's also new text for this section that needs review instead of this older stuff.